package.json

```diff
@@ -18,7 +18,8 @@
     "test": "lerna run test --npm-client=yarn --stream",
     "prettier:check": "prettier --check .",
     "prettier:write": "prettier --write .",
-    "prepublish": "yarn build && yarn copy-readme"
+    "prepublish": "yarn build && yarn copy-readme",
+    "patch:monaco": "cp ts.worker.patched.js node_modules/monaco-editor/esm/vs/language/typescript/ts.worker.js"
```

Comment:
What did you change in the ts.worker.js file? I would really rather not have a patch for monaco be part of the library, this will likely not work for most library consumers and make using the library really difficult, and will make this library become stale very quick as soon as monaco changes anything in that file.

Reply:
in `ts.worker.js` I've applied the [PR](https://github.com/microsoft/monaco-editor/pull/3057) proposed to monaco-editor to fix issue #7 
it is only for test/demo purpose and not affect the core distribution but you can remove it if have some concerns